contentgrid-spring-data-rest/src/test/java/com/contentgrid/spring/test/fixture/invoicing/InvoicingApplicationTests.java

```diff
@@ -899,11 +1485,13 @@ class OneToMany {
 
                 @Test
                 void deleteOrderById_fromInvoice_shouldReturn_http204() throws Exception {
-                    var invoice = invoices.findById(INVOICE_1_ID).orElseThrow();
-                    assertThat(invoice.getOrders()).contains(orders.findById(ORDER_1_ID).orElseThrow());
+                    doInTransaction(() -> {
+                        var invoice = invoices.findById(INVOICE_1_ID).orElseThrow();
+                        assertThat(invoice.getOrders()).contains(orders.findById(ORDER_1_ID).orElseThrow());
+                    });
```

Comment:
It's a bit strange that the check if the invoice contains an order is done before.

At least, the check that the order is removed from the invoice should be done afterwards